datalad/distribution/create.py

```diff
@@ -199,6 +199,13 @@ def __call__(
                                 git_opts=git_opts,
                                 annex_opts=annex_opts,
                                 annex_init_opts=annex_init_opts)
+                # record the annex uuid of this repo for this afterlife
+                # to be able to track siblings and children
+                if 'datalad.annex.origin' in ds.config:
+                    # make sure we reset this variable completely, in case of a re-create
+                    ds.config.unset('datalad.annex.origin', where='dataset')
+                ds.config.add('datalad.annex.origin', vcs.uuid, where='dataset')
+                ds.repo.add('.datalad', git=True)
```

Comment:
btw what happens to it when merging multiple datasets/repos? one to be chosen or a new one created or ... ?  just this tracking of the origin and siblings -- still not feeling where the benefit would come